### PR TITLE
w10: Almost done with analysis

### PR DESCRIPTION
**Pull request by C0DK**
### Description
Added text detailing how schools use the arduino platform in education

### Changelog
 - Added Arduino In Education
 - Added notes from the interview to misc

### Issues solved 
add these with the respective key as HCLANG-1 (as an example)
 - HCLANG-23

## FOR REVIEWER [Do not remove]
A small checklist of things to note when you are doing a review. Make sure these things apply before going into master.
 - Did the PR actually fix the issue
 - Comment anything that doesn't make sense or anything you want clarified. Many comments are always ok
 - Make sure this is ready for exams if this is going into master
 - It's always a good idea having another reviewer look at a PR as well, if it's major. get a buddy.

### Regarding filenames
- all .tex files use pascal case
- all other files [except in /code/] use snake case

### Regarding code [if applicable]
- Check [coding conventions](https://kotlinlang.org/docs/reference/coding-conventions.html)
- Are public methods documented? 
- Are there testcases for new features / methods?
- Have you tried running the source and tested things that cannot be test-cased?

### Regarding Report [if applicable]
- Does each line have a linechange in the sourcecode?
- Does sentences make sense?
- Did you do spellchecking?
- Citations?

### CODE REVIEW

**Comment by C0DK** [comment]
test

File: report/2.Analysis/ArduinoInEducation.tex [hunk 1]
```diff
@@ -0,0 +1,84 @@
+% point of document
```
Comment (Abildgaard96) [inline report/2.Analysis/ArduinoInEducation.tex]: Please remove comments when no longer needed

File: report/2.Analysis/ArduinoInEducation.tex [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+% We want to explain to the reader that (and how) the arduino platform is used in HTX
+% The point is:
+% - What "teknikfag" is.
+% - What "el-teknik" is.
+% - Document out findings of the student and teacher interviews.
+
+\section{Arduino and their usage in danish education}
```
Comment (Abildgaard96) [inline report/2.Analysis/ArduinoInEducation.tex]: and their usage? if Their referes to Arduino, it should be 'it's usage'

File: report/2.Analysis/ArduinoInEducation.tex [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+During the third year of the HTX\footnote{Higher Technical Examination Programme, is a 3-year vocationally oriented general upper secondary programme which builds on the 10th-11th form of the Folkeskole\cite{htx_wiki}} programme students are required to take a course in a voluntary technical subject, which utilizes a project-based learning practice. 
+There are a multiple of courses, one of these regard electronics, where many schools choose to utilize the Arduino platform\footnote{based on the fact that 4 of the students in this group studied at HTX in different parts of Denmark and all utilized the Arduino platform in this course. The HTX institution contacted by the group also utilize the Arduino platform}\cite{holstebro_education}.
```
Comment (Abildgaard96) [inline report/2.Analysis/ArduinoInEducation.tex]: delete 'of', 
should be "multiple courses,.."

File: report/2.Analysis/ArduinoInEducation.tex [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+The Programming course teaches the students the basic constructions and terminology of general programming. The students are taught C{\#} in the programming course. 
+
+The electronics course focuses on engaging the students in making arudino projects, not teaching the students the theory behind the projects. 
+This means that the students both have a very limited understanding of the arduino platform's architecture and a low understanding of the arduino language as explained earlier in the report, which they utilize in the electronics course on HTX CAO.
+
+Because of all this the electronics course has a relatively high learning curve. 
+Which results in a high number of students abandoning the course in favor of some of the courses with a more defined structures.
+
+Clearly there is a major gap between the students who are interested in programming or electronics and have no prior experience and students who have prior experiences. 
+Based on students' and the teacher's, statements it seems like the course need a better gateway into the arduino platform, the programming aspects of the course or both.
```
Comment (Abildgaard96) [inline report/2.Analysis/ArduinoInEducation.tex]: the ved ejefald.
'based on THE student's and...."

File: report/2.Analysis/ArduinoInEducation.tex [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+The selective control structures seems to be easier to understand than the iterative control structures. 
+A substantial amount of the students stated that the control structures in general are difficult to understand and remember, since they do not reflect anything else in the language, at least according to them.
+Clearly the students needs more consistency in the language. 
+The control structures should in some way mirror the syntax of another major aspect of the language, to make it more intuitive to use and remember.
+
+\textbf{Operator Precedence and Association}\\
+Both terms were completely foreign to the students. Their perception of precedence were mostly based on arithmetic notation. 
+This is sufficient in most cases. The biggest concern was with association. 
+Most of students did not understand that different operators could have different association, which proved to be a major issue when presented with code. 
+In most cases the students understood which part of the expression was calculated first.
```
Comment (Abildgaard96) [inline report/2.Analysis/ArduinoInEducation.tex]: Line 73 and 74 should be merged to ONE sentence, makes more sence

File: report/2.Analysis/ArduinoInEducation.tex [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+There are a multiple of courses, one of these regard electronics, where many schools choose to utilize the Arduino platform\footnote{based on the fact that 4 of the students in this group studied at HTX in different parts of Denmark and all utilized the Arduino platform in this course. The HTX institution contacted by the group also utilize the Arduino platform}\cite{holstebro_education}.
+
+This technically oriented course, often called electronics\footnote{The name varies between institutions}, aims to familiarize the students with electrical components and circuits as well as basic programming.\cite{holstebro_electronic}
+Arduino is an ideal platform for this, as it is possible to connect I/O devices to the Arduino micro-controller, which in turn can be programmed to do simple tasks. 
+This course is then used to spike the interest of the student regarding engineering fields, such as programming or electronics.
+
+
+\subsection{HTX CAO Electronic Course}
```
Comment (Abildgaard96) [inline report/2.Analysis/ArduinoInEducation.tex]: Maybe footnote for CAO

File: report/2.Analysis/ArduinoInEducation.tex [hunk 7]
```diff
@@ -0,0 +1,84 @@
+% point of document
+% We want to explain to the reader that (and how) the arduino platform is used in HTX
+% The point is:
+% - What "teknikfag" is.
+% - What "el-teknik" is.
+% - Document out findings of the student and teacher interviews.
+
+\section{Arduino and their usage in danish education}
+During the third year of the HTX\footnote{Higher Technical Examination Programme, is a 3-year vocationally oriented general upper secondary programme which builds on the 10th-11th form of the Folkeskole\cite{htx_wiki}} programme students are required to take a course in a voluntary technical subject, which utilizes a project-based learning practice.
```
Comment (Lambcreek) [inline report/2.Analysis/ArduinoInEducation.tex]: "Programme" is technically correct, however it is the British way of spelling it, and everywhere else we use the American way.
Change it to "Program"

File: report/1.Introduction/1.IntroductionCtrl.tex [hunk 8]
```diff
@@ -4,31 +4,29 @@ \chapter{Introduction}
...
 that are non-present in most programming languages.
 Although this is an advantage in terms of simplicity, it might halter, or even frustrate, the student when learning other programming languages.
 
-As such the motivation for this project is to create a new language for the Arduino platform, with a focus on promoting simplicity and the thought process that goes into programming, rather than difficult syntax and extraneous setup code.
+As such the motivation for this project is to create a new language for the Arduino platform, with a focus on promoting simplicity and teaching the thought process that goes into programming, rather than difficult syntax and extraneous setup code.
 
 %This project aims to create a new language for the Arduino platform that promotes simplicity and strives to teach student the fundamentals of programming.
 
-From this the following initial problem statement is drafted:
+Basedo on these considerations, the following initial problem statement has been drafted:
```
Comment (Abildgaard96) [inline report/1.Introduction/1.IntroductionCtrl.tex]: lul basedo

File: report/2.Analysis/ArduinoInEducation.tex (hunk 1 above)
Comment (C0DK) [reply report/2.Analysis/ArduinoInEducation.tex]: Var det ikke en regl at i toppen af alle dokumenter forklarede man kort pointen fra "vores" synsvinkel

File: report/2.Analysis/ArduinoInEducation.tex [hunk 9]
```diff
@@ -0,0 +1,88 @@
+% point of document
+% We want to explain to the reader that (and how) the arduino platform is used in HTX
+% The point is:
+% - What "teknikfag" is.
+% - What "el-teknik" is.
+% - Document out findings of the student and teacher interviews.
+
+\section{Arduino and it's usage in danish education}
```
Comment (Lambcreek) [inline report/2.Analysis/ArduinoInEducation.tex]: "it's" skal være "its"

File: report/2.Analysis/ArduinoInEducation.tex [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+During the third year of the HTX\footnote{Higher Technical Examination Program, is a 3-year vocationally oriented general upper secondary programme which builds on the 10th-11th form of the Folkeskole\cite{htx_wiki}} programme students are required to take a course in a voluntary technical subject, which utilizes a project-based learning practice.
```
Comment (Lambcreek) [inline report/2.Analysis/ArduinoInEducation.tex]: "Programme" skal være "Program". Vi bruger "program" alle andre steder i rapporten.

File: report/2.Analysis/ArduinoInEducation.tex (hunk 1 above)
Comment (nillas12) [reply report/2.Analysis/ArduinoInEducation.tex]: Jo. Så det skal blive, og skal reelt tilføjes til toppen af alle dokumenter hvor de mangler

File: report/2.Analysis/ArduinoInEducation.tex [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+Instead it focuses on project based arduino development. 
+Educational programming experience of the students (if any), originates from another optional course called "Programming". 
+The Programming course teaches the students the basic constructions and terminology of general programming. 
+The students are taught C{\#} in the programming course. 
+
+The electronics course focuses on engaging the students in making arudino projects, not teaching the students the theory behind the projects. 
+This means that the students both have a very limited understanding of the arduino platform's architecture and a low understanding of the arduino language as explained earlier in the report, which they utilize in the electronics course on Aalborg Tekniske Gymnasium.
+
+Because of all this the electronics course has a relatively high learning curve. 
+Which results in a high number of students abandoning the course in favor of some of the courses with a more defined structures.
```
Comment (Lambcreek) [inline report/2.Analysis/ArduinoInEducation.tex]: Enten skal "Which" ændres til "this", eller punktummet før skal laves om til komma.

File: report/2.Analysis/ArduinoInEducation.tex [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+
+Clearly there is a major gap between the students who are interested in programming or electronics and have no prior experience and students who have prior experiences. 
+Based on the students' and the teacher's statements it seems like the course need a better gateway into the arduino platform, the programming aspects of the course or both.
+
+A substantial number of the students in the class expressed their frustration with the relative complexity of the arduino language. 
+The teacher supplemented with saying that the language is not necessary complex, but the time spent on the different concepts is far too short, which is a result of the education being a junior education and not a college/university level education.
```
Comment (Lambcreek) [inline report/2.Analysis/ArduinoInEducation.tex]: "necessary" skal være "necessarily"

**Pull request merged**